lib/datadog/lambda.rb

```diff
@@ -85,15 +88,7 @@ def self.metric(name, value, time: nil, **tags)
       raise 'name must be a string' unless name.is_a?(String)
       raise 'value must be a number' unless value.is_a?(Numeric)
 
-      time ||= Time.now
-      time_ms = time.to_f.to_i
-
-      tag_list = ["dd_lambda_layer:datadog-ruby#{dd_lambda_layer_tag}"]
-      tags.each do |tag|
-        tag_list.push("#{tag[0]}:#{tag[1]}")
-      end
-      metric = { e: time_ms, m: name, t: tag_list, v: value }
-      puts metric.to_json
+      @metrics_client.distribution(name, value, time: time, **tags)
```

Comment:
Are these tests for this, file `lambda.rb`?
If so, these changes should be tested there.

Reply:
Resolved in latest commit